lib/utils/siprec-utils.js

```diff
@@ -47,8 +47,16 @@ const parseSiprecPayload = (req, logger) => {
       }
     }
 
-    if (!sdp || !meta) {
-      logger.info({payload: req.payload}, 'invalid SIPREC payload');
+
+    if (!meta && sdp) {
+      const arr = /^([^]+)(m=[^]+?)(m=[^]+?)$/.exec(sdp);
+      opts.sdp1 = `${arr[1]}${arr[2]}`;
+      opts.sdp2 = `${arr[1]}${arr[3]}\r\n`;
+      opts.sessionId = uuidv4();
+      logger.info({ payload: req.payload }, 'Cisco NBR payload');
```

Comment:
minor thing, but Cisco NBR might not be the only siprec payload that lacks metadata.  Can you modify the log statement to say something like "SIPREC payload with no metadata (e.g. Cisco NBR)"